### DISCUSSION TITLE
Zephyr platform Host-Processor support

### DISCUSSION THREAD

**Discussion by pdgendt** [openthread/openthread#7886] [Q&A]
Hi OpenThread Devs,

I'm working on [Host-Processor support in zephyr](https://github.com/zephyrproject-rtos/zephyr/pull/47256). I've started from the POSIX implementation, and kept some of the internal logic. 
As stated by the license headers, I kept those in the source files. However, this breaks compliance for zephyr as they only allow Apache 2 contributions.

So, my question, is it allowed to change the ported files to Apache 2?

**Comment by jwhui** [comment]
Is it possible for Zephyr to include the OpenThread code under the current license?  The [Zephyr contribution guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html) states:

> There are some imported or reused components of the Zephyr project that use other licensing, as described in [Licensing of Zephyr Project components](https://docs.zephyrproject.org/latest/LICENSING.html#zephyr-licensing).

The BSD license is fairly common and it may be fine for the Zephyr project to contain.
